docs/chi-examples/20-protobuf-schema-example.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: protobuf-schemas
+data:
+  your_schema.proto: |
+    syntax = "proto3";
+    package tutorial;
+    import "google/protobuf/timestamp.proto";
+
+    message Person {
+      string name = 1;
+      int32 id = 2;  // Unique ID number for this person.
+      string email = 3;
+
+      enum PhoneType {
+        MOBILE = 0;
+        HOME = 1;
+        WORK = 2;
+      }
+
+      message PhoneNumber {
+        string number = 1;
+        PhoneType type = 2;
+      }
+
+      repeated PhoneNumber phones = 4;
+
+      google.protobuf.Timestamp last_updated = 5;
+    }
+
+---
+apiVersion: "clickhouse.altinity.com/v1"
+kind: "ClickHouseInstallation"
+metadata:
+  name: "protobuf-example"
+spec:
+  configuration:
+    settings:
```

Comment:
@sunsingerus look like we have bug here
when i comment `settings` and run `kubectl apply`
I don't see regenerate event for configMap `chi-protobuf-example-common-configd`

Reply:
@sunsingerus when pod staying in CrashLoopBack state, then chi-{chi_name}-{cluster}-common-configd ConfigMap don't update